colmena.nix

```diff
@@ -1,6 +1,30 @@
 inputs:
 let
   targetSystem = "x86_64-linux";
+
+  oldGrubOnlySystem = {
+    boot.loader.grub = {
+      device = "/dev/sda";
+      efiSupport = false;
+      efiInstallAsRemovable = false;
+    };
+
+    fileSystems."/efi-boot".enable = false;
+  };
+
+  oldSystemdBootSystem = {
+    boot.loader = {
+      grub = {
+        enable = false;
+        efiInstallAsRemovable = false;
+      };
+      systemd-boot.enable = true;
+      efi = {
+        canTouchEfiVariables = true;
+        efiSysMountPoint = "/boot";
+      };
+    };
+  };
```

Comment:
Could you add a small comment explaining what kinds of systems we should expect to need to use either of these configs?